.github/workflows/operate-ci-build-reusable.yml

```diff
@@ -146,3 +147,12 @@ jobs:
         if: ${{ env.IS_DEFAULT_BRANCH == 'true' }}
         run: |
           mvn -f operate org.sonatype.plugins:nexus-staging-maven-plugin:deploy-staged -DskipTests=true -P -docker,skipFrontendBuild -B -DaltStagingDirectory=${{ github.workspace }}/staging-${{ env.BRANCH_NAME }} -DskipRemoteStaging=true -Dmaven.deploy.skip=true
+
+      #########################################################################
+      # Collect information about build status for central statistics with CI Analytics
+      - uses: camunda/infra-global-github-actions/submit-build-status@main
+        if: always()
+        with:
+          build_status: "${{ contains(steps.*.outcome, 'failure') && 'failure' || 'success' }}"
+          gcp_credentials_json: "${{ steps.secrets.outputs.gcloud_sa_key }}"
+          big_query_table_name: "ci-30-162810:prod_ci_analytics.build_status_v2"
```

Comment:
❓ Do we need some indentifier for the build, so it is clear that was operate or tasklist etc?

Reply:
This is done implicitly by the [action](https://github.com/camunda/infra-global-github-actions/tree/main/submit-build-status#behavior) from environment variables using the workflow name, job name, etcpp:

![image](https://github.com/camunda/zeebe/assets/800933/caf33cba-18ea-4668-9e8d-c2a36f344fd3)

Reply:
I will look at data visualization later :)